Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -187,22 +203,148 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
+      Memory::Write_U32(0x0a400001 | crc << 1, _BufferOut + 12);
+    }
+    else
+    {
+      // 2048 bytes/sector
+      u32 read_bl_len = 11;
+
+      // size = (c_size + 1) * (1 << (2 + c_size_mult + read_bl_len))
+      u64 c_size = size;
+      u32 c_size_mult = 0;
+      bool invalid_size = false;
+      while (c_size > 4096)
+      {
+        invalid_size |= c_size & 1;
+        c_size >>= 1;
+        if (++c_size_mult >= 8 + 2 + read_bl_len)
+        {
+          ERROR_LOG(IOS_SD, "SD Card is too big!");
+          // Set max values
+          c_size = 4096;
+          c_size_mult = 7 + 2 + read_bl_len;
+        }
+      }
+      c_size_mult -= 2 + read_bl_len;
+      --c_size;
+
+      if (invalid_size)
+        WARN_LOG(IOS_SD, "SD Card size is invalid");
+      else
+        INFO_LOG(IOS_SD, "SD C_SIZE = %lu, C_SIZE_MULT = %u", c_size, c_size_mult);
+
+      // 0b00           CSD_STRUCTURE (SDv1)
+      // 0b000000       reserved
+      // 0b01111111     TAAC (8.0 * 10ms)
+      // 0b00000000     NSAC
+      // 0b00110010     TRAN_SPEED (2.5 * 10 Mbit/s, max operating frequency)
+
+      // 0b010110110101 CCC
+      // 0b1111         READ_BL_LEN (2048 bytes)
+      // 0b1            READ_BL_PARTIAL
+      // 0b0            WRITE_BL_MISALIGN
+      // 0b0            READ_BLK_MISALIGN
+      // 0b0            DSR_IMP (no driver stage register implemented)
+      // 0b00           reserved
+      // 0b??????????   C_SIZE (most significant 10 bits)
+
+      // 0b??           C_SIZE (least significant 2 bits)
+      // 0b111          VDD_R_CURR_MIN (100 mA)
+      // 0b111          VDD_R_CURR_MAX (100 mA)
+      // 0b111          VDD_W_CURR_MIN (100 mA)
+      // 0b111          VDD_W_CURR_MAX (100 mA)
+      // 0b???          C_SIZE_MULT
+      // 0b1            ERASE_BLK_EN (erase unit = 512 bytes)
+      // 0b1111111      SECTOR_SIZE (128 write blocks)
+      // 0b0000000      WP_GRP_SIZE
+
+      // 0b0            WP_GRP_ENABLE (no write protection)
+      // 0b00           reserved
+      // 0b001          R2W_FACTOR (write half as fast as read)
+      // 0b1111         WRITE_BL_LEN (= READ_BL_LEN)
+      // 0b0            WRITE_BL_PARTIAL (no partial block writes)
+      // 0b00000        reserved
+      // 0b0            FILE_FORMAT_GRP (default)
+      // 0b1            COPY (contents are copied)
+      // 0b0            PERM_WRITE_PROTECT (not permanently write protected)
+      // 0b0            TMP_READ_PROTECT (not temporarily write protected)
+      // 0b00           FILE_FORMAT (contains partition table)
+      // 0b00           reserved
+      // 0b???????      CRC
+      // 0b1            reserved
+
+      // TODO: CRC7
+      u32 crc = 0;
+
+      Memory::Write_U32(0x007f0032, _BufferOut);
+      Memory::Write_U32(0x5b5f8000 | (c_size >> 2), _BufferOut + 4);
+      Memory::Write_U32(0x3ffc7f80 | (c_size << 30) | (c_size_mult << 15), _BufferOut + 8);
+      Memory::Write_U32(0x07c04001 | (crc << 1), _BufferOut + 12);
```

Comment:
Same down here, if you actually know what those values are and where they're from/what they should be set to.